hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/utils/autoReloadHelper.tsx

```diff
@@ -49,6 +49,10 @@ class AutoReloadHelper {
       this.stopPolling();
     }
   };
+
+  toggleChecking = (checked: boolean) => {
+    sessionStorage.setItem('toggleCheck', JSON.stringify(checked));
+  };
```

Comment:
Couldn't we move this `sessionStorage.setItem('toggleCheck', JSON.stringify(checked));` line into the already existing `handleAutoReloadToggle` method? For example to the first line of the method, before the conditions. If we do it this way we wouldn't need a new `toggleChecking` method.

Reply:
hmm.. for myself learning, what's the difference between _checked_ in method _handleAutoReloadToggle_, and _toggleCheck_ in new method _toggleChecking_? 
only type difference??

Reply:
There is basically no difference, I just thought that with the approach I suggested we can avoid adding and passing another method between the pages with the same parameter. Also I think this one line that is in the method fits in the already existing one, the naming isn't confusing either, the functionality of `handleAutoReloadToggle` should be handling that toggle, so that storage setting is okay there I think.

Reply:
Ohh I think I misunderstood you question @DaveTeng0, I thought you are asking about the change I suggested, sorry :) `toggleCheck` will be the property we set in the [sessionStorage](https://developer.mozilla.org/en-US/docs/Web/API/Window/sessionStorage), so in the browser we can ask for that property until the page session ends (if we would set it in the [localStorage](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage) that would have no expiration time). The `checked`  is a variable in the code, which stores the current state of the toggle in the Recon UI. That is why in the new method we set the `toggleCheck` to `checked`, which is passed as a parameter to the method (same parameter is used in the already existing `handleAutoReloadToggle` method). I hope this answers your question :) correct me if I am wrong @smitajoshi12

Reply:
Make sense! So, is it possible that we could use the var `checked` in sessionStorage with its original name, instead of creating a new var `toggleCheck` ?

Reply:
What we could do is to name the property that is set in the sessionStorage to `checked`, but as we have this information in the browser it is better to have a name that is easy to understand what it stands for.